testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/jboss-ejb3.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jboss:jboss
```

Comment:
I think the root element should be jboss:ejb-jar. It was there before but not sure how if worked, probably was not validated.